### DISCUSSION TITLE
New Dashbaords Upload/Download FEATURE FLAG

### DISCUSSION THREAD

**Discussion by reinhard-brandstaedter** [Dynatrace/dynatrace-configuration-as-code#1513] [General]
Hi,
just learned that monaco has [feature flags that would support uploading/downloading new dashboards](https://community.dynatrace.com/t5/Dashboarding/Downloading-new-Dashboards-with-Monaco/m-p/246783/highlight/true#M3992).
Is there a documentation how to enable features via flags? How would I do this e.g. for dashboards?

**Comment by UnseenWizzard** [comment]
Hi @reinhard-brandstaedter, 
Monaco generally uses feature-flags to allow us to build new features while still being able to release bugfixes without the need to backport fixes to the latest release version/branch.

As such, they are currently purposely not documented and only described internally for early feedback and testing of new features - and things might still change while a feature is not active.
A handful of flags, you can find described in the documentation, like the optional ['fast (but memory inefficient) dependency resolver' for downloads](https://docs.dynatrace.com/docs/shortlink/configuration-as-code-commands#dependency-resolution)

We have discussed publishing a list of feature flags in the past, and will revisit that.

Support for new Dashboards and Notebooks is upcoming in the next 2.15.0 version, which is planned to release in the next weeks.

If you're feeling adventurous, take a look in the repo for the `featureflags` package ;) 
They're all turned on or off as described in that doc link above, by setting their environment variable to `true` (or `1`) or `false` (`0`).
**But be advised that changing flags will mean support is not able to help you with any issues, especially for features that have not released yet.**
